app/models/vacancy.rb

```diff
@@ -46,7 +46,7 @@ class Vacancy < ApplicationRecord
   array_enum phases: { nursery: 0, primary: 1, middle: 2, secondary: 3, sixth_form_or_college: 4, through: 5 }
   array_enum job_roles: JOB_ROLES
 
-  enum contract_type: { permanent: 0, fixed_term: 1, parental_leave_cover: 2 }
+  enum contract_type: { permanent: 0, fixed_term: 1, casual: 3 }
```

Comment:
Reading the rake task,  it looks like this removal needs to go in a follow-up PR, right?

Since it is referenced here:

```ruby
vacancies_to_update = Vacancy.where(contract_type: Vacancy.contract_types[:parental_leave_cover])
```